src/vsphere_cpi/lib/cloud/vsphere/vm_type.rb

```diff
@@ -14,9 +14,12 @@ def initialize(datacenter, cloud_properties, pbm)
       @pbm = pbm
     end
 
-    %w[cpu ram disk cpu_hot_add_enabled memory_hot_add_enabled nsx vmx_options vgpus
-       nsxt memory_reservation_locked_to_max nested_hardware_virtualization
-       upgrade_hw_version datacenters vm_group disable_drs storage_policy tags].each do |name|
+    %w[cpu cpu_hot_add_enabled datacenters disable_drs disk
+       memory_hot_add_enabled memory_reservation_locked_to_max
+       nested_hardware_virtualization nsx nsxt pci_passthroughs ram
+       storage_policy tags upgrade_hw_version vgpus vm_group
+       vmx_options
+    ].each do |name|
```

Comment:
I realize the change here is only adding to an existing pattern, I think there might be a nice refactor to consider here where the list of "things to define as methods" is pulled out to a constant, and each "method" lives on its own line.

Something like: 
```ruby
CLOUD_PROPERTIES_HASH_PROXY_METHODS = [
  :cpu,
  :cpu_hot_add_enabled,
  :datacenters,
  :disable_drs,
  :disk,
  :memory_hot_add_enabled,
  :memory_reservation_locked_to_max,
  :nested_hardware_virtualization,
  :nsx,
  :nsxt,
  :pci_passthroughs,
  :ram,
  :storage_policy,
  :tags,
  :upgrade_hw_version,
  :vgpus,
  :vm_group,
  :vmx_options,
]

CLOUD_PROPERTIES_HASH_PROXY_METHODS.each do |method_name|
  define_method(name) { @cloud_properties[name] }
end
```

This would make future diff's for adding/removing methods clearer, and the code would likely be easier to scan.

Reply:
Great idea! Done.